opencog/nlp/aiml/aiml.scm

```diff
@@ -265,7 +265,7 @@
 ; XXX Need to handle that, topic rules as appropriate.
 (define (run-pattern-rule RULE SENT)
 	(define maplk (MapLink
-		(ImplicationLink
+		(ImplicationScopeLink
 			(gdr (get-pred RULE "*-AIML-pattern-*"))
```

Comment:
I'm pretty sure this use is not meant to be scoped.  I think that this is supposed to be the same as what openpsi uses, which is not scoped.